plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergConfig.java

```diff
@@ -336,4 +337,17 @@ public IcebergConfig setMaterializedViewsStorageSchema(String materializedViewsS
         this.materializedViewsStorageSchema = Optional.ofNullable(materializedViewsStorageSchema);
         return this;
     }
+
+    public boolean isSortedWritingEnabled()
+    {
+        return sortedWritingEnabled;
+    }
+
+    @Config("iceberg.sorted-writing-enabled")
```

Comment:
It can also be useful if your writes are very small (streaming ingest, for example) such that sorting them would be a waste of time until they are compacted.